### ISSUE TITLE
Couldn't run by old taskbar's icon when installed newer version

### ISSUE THREAD

**Issue by mystasly48** [mystasly48/Mystter_SendTweet#27]
Here is the step to reproduce the issue.
1. Install older version (v1.4.0 for example),
1. Run the program by clicking start menu shortcut.
1. Pin to taskbar.
1. Install newer version (v1.4.1 for example).
1. Run the program by clicking the taskbar icon.

Actually I don't know if this is a bug. But I believe that this can be solved by changing some property of the installer project.

**Comment by mystasly48** [comment]
I haven't distributed v1.4.1 or newer version of the installer so nobody can reproduce the issue without rebuilding...

**Comment by mystasly48** [comment]
I found the way to solve this issue manually. Basically the target location of the shortcuts in the taskbar or start menu are set to the application file.
For example, target location of the shortcut of Firefox is set for `"C:\Program Files (x86)\Mozilla Firefox\firefox.exe"`.
But the target location of my shortcut is set for `Mystter`.
When I created a shortcut that the target location is `C:`Program Files (x86)\Maisu\Mystter\Mystter.exe`, I could run by taskbar icon even if I updated the program.
Actually I don't know why the shortcut works normally, but I'm sure that this is the reason.
